src/Reflection/InitializerExprTypeResolver.php

```diff
@@ -928,8 +932,10 @@ public function getPlusType(Expr $left, Expr $right, callable $getTypeCallback):
 		}
 
 		$arrayType = new ArrayType(new MixedType(), new MixedType());
+		$leftIsArraySuperType = $arrayType->isSuperTypeOf($leftType);
+		$rightIsArraySuperType = $arrayType->isSuperTypeOf($rightType);
```

Comment:
you should be able to use isArray here now instead

Reply:
one of is going to have a conflict here :)

Reply:
resolved ;)